models/openmc/make_beavrs.py

```diff
@@ -16,6 +16,12 @@
 p.add_argument('-s', '--symmetric', action='store_true', dest='is_symmetric', \
                default=False, help='Create octant-symmetric input files,' \
                + ' not symmetric by default.')
+p.add_argument('-i', '--sym-instrument', action='store_true', dest='sym_instrument', \
+               default=False, help='Add instrument tubes to all assemblies in an ' \
+               + 'octant-symmetric input file (only valid if the model is symmetric).' \
+               + 'Defaults to not adding instrument tubes per-assembly. This makes it '
```

Comment:
"instrument tubes per-assembly" sounds awkward. I think this sentence can be removed from the description